src/api/github/getClient.ts

```diff
@@ -63,15 +58,20 @@ export async function getClient(type: ClientType, org: string | null) {
       );
     }
 
+    const auth: AppAuthStrategyOptions = {
+      appId: Number(process.env.GH_APP_IDENTIFIER),
+      privateKey: process.env.GH_APP_SECRET_KEY?.replace(/\\n/g, '\n'),
+    };
+
     let client = _CLIENTS_BY_ORG.get(org);
     if (client === undefined) {
-
       // Bootstrap with a client not bound to an org.
-      const appClient = _getAppClient();
+      const appClient = _getAppClient(auth);
 
       // Use the unbound client to hydrate a client bound to an org.
       const installation = await appClient.apps.getOrgInstallation({ org });
-      client = _getAppClient(installation.data.id);
+      auth.installationId = installation.data.id;
```

Comment:
I logged locally and also browsed GitHub's UI for existing apps, this does appear to be numeric (per the type def above).